extensions/git/package.json

```diff
@@ -1922,6 +1922,33 @@
           "highContrast": "#E2C08D"
         }
       },
+      {
+        "id": "gitDecoration.stageAddedResourceForeground",
+        "description": "%colors.stageAdded%",
+        "defaults": {
+          "light": "#0a7a62",
+          "dark": "#02c39a",
+          "highContrast": "#00cea1"
+        }
+      },
+      {
+        "id": "gitDecoration.stageModifiedResourceForeground",
+        "description": "%colors.stageModified%",
+        "defaults": {
+          "light": "#FF661F",
+          "dark": "#FFAB85",
+          "highContrast": "#ca541d"
+        }
+      },
+      {
+        "id": "gitDecoration.stageDeletedResourceForeground",
+        "description": "%colors.stageDeleted%",
+        "defaults": {
+          "light": "#ad0707",
+          "dark": "#c74e39",
+          "highContrast": "#c74e39"
```

Comment:
I don't think we should change the default colors, so let's just use today's colors. This is about enabling themes to overwrite this, not to change the default themes.